lib/charms/layer/basic.py

```diff
@@ -69,7 +69,6 @@ def bootstrap_charm_deps():
             # (see https://github.com/pypa/pip/issues/410)
             fp.writelines([
                 "[easy_install]\n",
-                "allow_hosts = ''\n",
                 "find_links = file://{}/wheelhouse/\n".format(charm_dir),
             ])
```

Comment:
What about this instead:

```suggestion
        if series in ('ubuntu12.04', 'precise',
                          'ubuntu14.04', 'trusty',
                          'ubuntu16.04', 'xenial',
                          'ubuntu18.04', 'bionic'):
            with open('/root/.pydistutils.cfg', 'w') as fp:
                # make sure that easy_install also only uses the wheelhouse
                # (see https://github.com/pypa/pip/issues/410)
                fp.writelines([
                    "[easy_install]\n",
                    "allow_hosts = ''\n",
                    "find_links = file://{}/wheelhouse/\n".format(charm_dir),
                ])```